setup.py

```diff
@@ -28,10 +28,11 @@
 # README file as long_description.
 long_description = io.open('README.rst', encoding='utf-8').read()
 
-# If CIRQ_UNSTABLE_VERSION is set then we use cirq-unstable as the name of the package
-# and update the version to this value.
+# If CIRQ_UNSTABLE_VERSION is set then we update the version to this value.
+# As it contains "dev" it will be a pre-release version on PyPi. See
```

Comment:
IIUC, this should be changed: "As it contains" -> "If it contains"

("As" implies we know it does, which I don't think we do.)

Reply:
We can clarify that - but it _is_ assumed that it contains dev.

Reply:
Actually it can contain whatever, but it is assumed that it contains `devN`, `aN`, `bN`, `rcN` hence the long description that changes it to the warning text below